### PR TITLE
rm .FTP.php.swp file

### PR DESCRIPTION
**Pull request by fransallen**

### CODE REVIEW

File: wp2static-addon-ftp.php [hunk 1]
```diff
@@ -55,7 +55,6 @@
 function run_wp2static_addon_ftp() {
     $plugin = new WP2Static\FTPAddon();
     $plugin->run();
-
 }
 
-run_wp2static_addon_ftp();
+run_wp2static_addon_ftp();
```
Comment (leonstafford) [inline wp2static-addon-ftp.php:60]: We should keep a newline at the end of each file as best practice. 

I pushed the swp file removal before this, so you should be able to pull latest and revise this with another commit

Comment (fransallen) [reply wp2static-addon-ftp.php:60]: Perfect!
